common/src/main/java/net/opentsdb/collections/LongLongHashTable.java

```diff
@@ -78,16 +154,21 @@ public void close() {
   }
 
   private void resize() {
-
+    long start = System.nanoTime();
+    int oldLength = arrayLength;
     int oldCap = slots;
     int newCap = oldCap;
     if (table != null) { // growing the table;
-      newCap = oldCap << 1; // double the size;
+      newCap = (int) (oldCap + (oldCap * growthRate));
```

Comment:
Now, after the table is grown, the new `size` may not be a prime. In such case the modulo hash `(key % size)` can cause collisions. That might be okay for now, but something to keep in mind.

Reply:
True, we have to keep an eye on that.